src/object/srv_obj.c

```diff
@@ -125,7 +125,9 @@ obj_rw_complete(crt_rpc_t *rpc, unsigned int map_version,
 		}
 
 		if (rc != 0) {
-			D_CDEBUG(rc == -DER_REC2BIG, DLOG_DBG, DLOG_ERR,
+			D_CDEBUG(rc == -DER_REC2BIG || rc == -DER_INPROGRESS ||
```

Comment:
I have a VOS_TX_LOG_* that essentially does these checks...perhaps we could do something generic like that here to avoid the code duplication

Reply:
Yes, we need some code cleanup for our current log messages. There are many misguided/repeated logs in current implementation.